be/src/exec/broker_reader.cpp

```diff
@@ -112,24 +112,29 @@ Status BrokerReader::open() {
         LOG(WARNING) << ss.str();
         return Status(ss.str());
     }
-
+    _file_size = response.size;
```

Comment:
for compatible with old broker, you should check if there is size field in response
if (reponse.__isset.size) {

}